### PR TITLE
[SOL-1856] User is taken back to coupons home on clicking next button…

### PR DESCRIPTION
**Pull request by iivic**
When clicked next, previous or active button nothing should happen. @marjev @vkaracic

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @iivic! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

To automatically create an OSPR issue for this pull request, just visit this link: https://openedx-webhooks.herokuapp.com/github/process_pr?number=838&repo=edx%2Fecommerce

**Comment by vkaracic** [comment]
You are missing a test for this.

**Comment by iivic** [comment]
Comment addressed.

File: ecommerce/static/js/views/dynamic_catalog_view.js [hunk 1]
```diff
@@ -24,6 +25,10 @@ define(['jquery',
                 this._super();
             },
 
+            stopEventPropagation: function(event) {
```
Comment (marjev) [inline ecommerce/static/js/views/dynamic_catalog_view.js:28]: You never call this function in tests.

Comment (iivic) [inline ecommerce/static/js/views/dynamic_catalog_view.js:28]: Done.

**Comment by vkaracic** [comment]
:+1:

**Comment by marjev** [comment]
👍

**Pull request merged**